practice-two/src/styles/components/text.css

```diff
@@ -27,3 +27,42 @@ li {
 .text-bold {
   font-weight: var(--fw-bold);
 }
+
+.text-sm {
+  font-size: 14px;
+  line-height: 34px;
+}
+
+.text-md {
+  font-size: 16px;
+  font-weight: var(--fw-normal);
+  line-height: 26px;
+}
+
+.text-lg {
+  font-size: 24px;
+  line-height: 34px;
+}
+
+.text-xl {
+  font-size: 36px;
+  line-height: 34px;
+  font-weight: var(--fw-bold);
```

Comment:
.text class? No need to use .text class?

Reply:
to reuse the same texts